### PR TITLE
chore: updating to use Uno.Fonts.Fluent package

### LINKED ISSUE #729: Update to use Uno.Fonts.Fluent package
Following the [guidelines here](https://github.com/unoplatform/uno.fonts#for-unofontsfluent) we should be installing the [Uno.Fonts.Fluent](https://www.nuget.org/packages/Uno.Fonts.Fluent) package into the Gallery app heads and replacing the old way that we were loading the uno-fluentui-assets.ttf font files.

### PR DESCRIPTION
**Pull request by rajamatt**
GitHub Issue (If applicable): closes #729 

## PR Type

What kind of change does this PR introduce?

- Build and CI
  - Change from using uno-fluentui-assets.ttf files to using Uno.Fonts.Fluent

## What is the current behavior?

Fluent fonts now use NuGet package instead of .ttf files.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested on iOS.
- [x] Tested on Wasm.
- [x] Tested on Android.
- [x] Tested on UWP.
- [x] Tested in both **Light** and **Dark** themes.
- [x] Associated with an issue (GitHub or internal)

### CODE REVIEW

**Review by agneszitte — APPROVED** [review]

File: Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems [hunk 1]
```diff
@@ -64,7 +64,6 @@
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Light.ttf" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Medium.ttf" />
     <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\Material\Roboto-Regular.ttf" />
-    <Content Include="$(MSBuildThisFileDirectory)Assets\Fonts\uno-fluentui-assets.ttf" />
```
Comment (Xiaoy312) [inline Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems:67]: should delete that file too

**Review by Xiaoy312 — COMMENTED** [review]
just do a quick global search for: uno-fluentui-assets.ttf
and also check if there is any file named like that

File: Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems (hunk 1 above)
Comment (Xiaoy312) [reply Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems:67]: https://github.com/unoplatform/Uno.Gallery/blob/dev/mara/update-icon-font/Uno.Gallery/Uno.Gallery.Shared/Assets/Fonts/uno-fluentui-assets.ttf

Comment (rajamatt) [reply Uno.Gallery/Uno.Gallery.Shared/Uno.Gallery.Shared.projitems:67]: Missed that one thanks! Search doesn't bring up any other results

**Pull request merged**